src/Microsoft.SqlTools.ServiceLayer/ExecutionPlan/ExecutionPlanGraphUtils.cs

```diff
@@ -163,6 +164,173 @@ public static List<ExecutionPlanGraphPropertyBase> GetProperties(PropertyDescrip
+            {
+                result.Add(new TopOperationsDataItem
+                {
+                    ColumnName = SR.Object,
+                    DataType = PropertyValueDataType.String,
+                    DisplayValue =  ((ExpandableObjectWrapper)currentNode["Object"]).DisplayName
+                });
+            }
+
+            result.Add(new TopOperationsDataItem
+            {
+                ColumnName = SR.EstimatedCost,
+                DataType = PropertyValueDataType.Number,
+                DisplayValue = Math.Round(currentNode.Cost, 1)
+            });
+
+            result.Add(new TopOperationsDataItem
+            {
+                ColumnName = SR.EstimatedSubtree,
+                DataType = PropertyValueDataType.Number,
+                DisplayValue = Math.Round(currentNode.SubtreeCost, 1)
+            });
+
+            if (currentNode["ActualRows"] != null)
+            {
+                result.Add(new TopOperationsDataItem
+                {
+                    ColumnName = SR.ActualRows,
+                    DataType = PropertyValueDataType.Number,
+                    DisplayValue = currentNode["ActualRows"]
+                });
+            }
+
+            if (currentNode["EstimatedDataSize"] != null)
+            {
+                result.Add(new TopOperationsDataItem
+                {
+                    ColumnName = SR.EstimatedRows,
+                    DataType = PropertyValueDataType.Number,
+                    DisplayValue = currentNode["EstimatedDataSize"]
+                });
+            }
+
+            if (currentNode["ActualExecutions"] != null)
+            {
+                result.Add(new TopOperationsDataItem
+                {
+                    ColumnName = SR.ActualExecutions,
+                    DataType = PropertyValueDataType.Number,
+                    DisplayValue = currentNode["ActualExecutions"]
+                });
+            }
+
+            if (currentNode["EstimateExecutions"] != null)
+            {
+                result.Add(new TopOperationsDataItem
+                {
+                    ColumnName = SR.EstimatedExecutions,
+                    DataType = PropertyValueDataType.Number,
+                    DisplayValue = currentNode["EstimateExecutions"]
+                });
+            }
+
+            if (currentNode["EstimateCPU"] != null)
+            {
+                result.Add(new TopOperationsDataItem
+                {
+                    ColumnName = SR.EstimatedCpu,
+                    DataType = PropertyValueDataType.Number,
+                    DisplayValue = currentNode["EstimateCPU"]
+                });
+            }
+
+            if (currentNode["EstimateIO"] != null)
+            {
+                result.Add(new TopOperationsDataItem
+                {
+                    ColumnName = SR.EstimatedIO,
+                    DataType = PropertyValueDataType.Number,
+                    DisplayValue = currentNode["EstimateIO"]
```

Comment:
Same as aboce

Reply:
Unfortunately, these properties names are coming from the engine and there's not much we can do on our end. If you read a sqlplan file you should see these properties named like this.